samples/Nim/nim.cfg

```diff
@@ -0,0 +1,270 @@
+# from https://github.com/nim-lang/Nim/blob/devel/config/nim.cfg
```

Comment:
This one file should be in `samples/Nim/filenames/` since you added it as a filename.

Reply:
done